### ISSUE TITLE
You didn't use the loss function introduced in the original paper

### ISSUE THREAD

**Issue by mohammad69h94** [sayakpaul/Supervised-Contrastive-Learning-in-TensorFlow-2#9]
You didn't use the loss function introduced in the original paper. Do you know how I can use that? Do you have it in a special place?
